src/server/applications/index.njk

```diff
@@ -67,7 +67,17 @@
                   html: appCaseListTab({
                       tabLabel: "Team cases",
                       tabTestId: "team-cases-tab",
-                      tableHeaders: tableHeaders,
+                      tableHeaders: [
```

Comment:
better to have them as before right so we can have reuse. If you need to pass in the suffix should be able to do that.